src/utilities/transition/ExpandableTransition.js

```diff
@@ -0,0 +1,113 @@
+'use strict';
+
+// Required modules.
+var Events = require( '../../mixins/Events.js' );
+var BaseTransition = require( './BaseTransition' );
+var fnBind = require( '../function-bind' ).bind;
+var contains = require( '../dom-class-list' ).contains;
+
+// Exported constants.
+var CLASSES = {
+  BASE_CLASS:     'u-expandable-transition',
+  EXPANDED:       'u-expandable-expanded',
+  COLLAPSED:      'u-expandable-collapsed',
+  OPEN_DEFAULT:   'u-expandable-content__onload-open'
+};
+
+/**
+ * ExpandableTransition
+ * @class
+ *
+ * @classdesc Initializes new ExpandableTransition behavior.
+ *
+ * @param {HTMLNode} element
+ *   DOM element to apply move transition to.
+ * @returns {ExpandableTransition} An instance.
+ */
+function ExpandableTransition( element, CLASSES ) { // eslint-disable-line max-statements, no-inline-comments, max-len
```

Comment:
`CLASSES` implies a constant, but an argument will never be guaranteed constant since it will be set to whatever value is passed in when the function is called.